shunting-yard-common/src/main/java/com/hotels/shunting/yard/common/metrics/MetricsConstant.java

```diff
@@ -17,10 +17,8 @@
 
 public final class MetricsConstant {
 
-  public static final String EMITTER_FAILURES = "shunting_yard_emitter_failures";
-  public static final String EMITTER_SUCCESSES = "shunting_yard_emitter_successes";
-
   public static final String RECEIVER_FAILURES = "shunting_yard_receiver_failures";
+  public static final String RECEIVER_EMPTY = "shunting_yard_receiver_empty";
```

Comment:
I figured we may as well count these so we can plot how many times they happen, should probably "never" happen since the `SqsMessageReader` sits in a loop waiting for messages to arrive before returning the call to `next()`.